cocotb/share/makefiles/simulators/Makefile.questa

```diff
@@ -43,7 +43,7 @@ else
     export MODELSIM_BIN_DIR
 endif
 
-RTL_LIBRARY ?= work
+RTL_LIBRARY ?= sim_build/work
```

Comment:
RTL_LIBRARY is expected to be a library name, not a path (that's what we use for other simulators as well). So this needs to stay "work".

Reply:
Ah, okay. I think that is fine; it'll just require rewriting runsim.do to use ```${SIM_BUILD}/${RTL_LIBRARY}``` everywhere (instead of just ```${RTL_LIBRARY}```) I guess.

Is that an okay solution? The library gets passed to questa via the ```-work``` flag, which seems to be quite happy taking a path (hence why this commit works).

(Of course, if we got rid of sim_build here then that wouldn't be necessary, as discussed in the other comment).

Reply:
Sorry for the delay here. I've pushed a new commit making this change; RTL_LIBRARY is still "work" and references to it in the makefile use SIM_BUILD/RTL_LIBRARY rather than just RTL_LIBRARY.